### PR TITLE
[zh]update file content/zh/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you.md

### PR DESCRIPTION
**Pull request by kinzhi**

### CODE REVIEW

File: content/zh/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you.md [hunk 1]
```diff
@@ -171,3 +174,10 @@ by Kubernetes.
 但用 Docker 创建、下载的镜像，对于容器运行时和 Kubernetes，均不可见。
 为了在 Kubernetes 中使用，需要把镜像推送（push）到某镜像仓库。
 
+<!-- ## {{% heading "whatsnext" %}}
+
+- Read [Migrating from dockershim](/docs/tasks/administer-cluster/migrating-from-dockershim/) to understand your next steps
+- Read the [dockershim deprecation FAQ](/blog/2020/12/02/dockershim-faq/) article for more information. -->
+## {{% 标题 "下一步" %}}
+- 阅读 [从 dockershim 迁移](/zh/docs/tasks/administer-cluster/migrating-from-dockershim/) 以了解您的后续步骤
```
Comment (jihoon-seo) [inline content/zh/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you.md:182]: [Netlify build fails](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/625ee5e153cdb60008987b88#L77).

```suggestion
## {{% heading "whatsnext" %}}

- 阅读 [从 dockershim 迁移](/zh/docs/tasks/administer-cluster/migrating-from-dockershim/) 以了解您的后续步骤
```

**Comment by kinzhi** [comment]
@jihoon-seo The document has been perfected, please help to review it again, thanks

File: content/zh/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you.md [hunk 2]
```diff
@@ -171,3 +174,11 @@ by Kubernetes.
 但用 Docker 创建、下载的镜像，对于容器运行时和 Kubernetes，均不可见。
 为了在 Kubernetes 中使用，需要把镜像推送（push）到某镜像仓库。
 
+<!-- ## {{% heading "whatsnext" %}}
+
+- Read [Migrating from dockershim](/docs/tasks/administer-cluster/migrating-from-dockershim/) to understand your next steps
+- Read the [dockershim deprecation FAQ](/blog/2020/12/02/dockershim-faq/) article for more information. -->
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you.md:180]:
```suggestion
- Read the [dockershim deprecation FAQ](/blog/2020/12/02/dockershim-faq/) article for more information.
-->
```

File: content/zh/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you.md [hunk 3]
```diff
@@ -171,3 +174,11 @@ by Kubernetes.
...
+## {{% heading "whatsnext" %}}
+
+- 阅读 [从 dockershim 迁移](/zh/docs/tasks/administer-cluster/migrating-from-dockershim/) 以了解您的后续步骤
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you.md:183]:
```suggestion
- 阅读[从 dockershim 迁移](/zh/docs/tasks/administer-cluster/migrating-from-dockershim/)以了解你的下一步工作
```

File: content/zh/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you.md [hunk 4]
```diff
@@ -171,3 +174,11 @@ by Kubernetes.
...
+- 阅读 [dockershim 弃用常见问题解答](/zh/blog/2020/12/02/dockershim-faq/) 文章了解更多信息。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you.md:184]:
```suggestion
- 阅读 [dockershim 弃用常见问题解答](/zh/blog/2020/12/02/dockershim-faq/)文章了解更多信息。
```

File: content/zh/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you.md [hunk 5]
```diff
@@ -171,3 +174,12 @@ by Kubernetes.
...
 为了在 Kubernetes 中使用，需要把镜像推送（push）到某镜像仓库。
 
+<!-- ## {{% heading "whatsnext" %}}
+
+- Read [Migrating from dockershim](/docs/tasks/administer-cluster/migrating-from-dockershim/) to understand your next steps
+- Read the [dockershim deprecation FAQ](/blog/2020/12/02/dockershim-faq/) article for more information. 
+-->
+## {{% heading "whatsnext" %}}
+
+- 阅读 [从 dockershim 迁移](/zh/docs/tasks/administer-cluster/migrating-from-dockershim/) 以了解你的下一步工作
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you.md:184]:
```suggestion
- 阅读[从 dockershim 迁移](/zh/docs/tasks/administer-cluster/migrating-from-dockershim/) 以了解你的下一步工作
```

**Pull request merged**